### ISSUE TITLE
[Enhance] Spark Accordion Item will use <button> as their triggering elements - React

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#2940]
**User Story:**
As Spark we want to update Spark Accordions from a link to `<button>` so that it is accessible.

**Notes:**
Currently spark's accordions use `<a>` when, since they don't navigate, should be `<button>`. RM-12 in the a11y audit.

**AC:**
React Accordion
1. will replace `<a>` with `<button>`
2. unit test will pass

**Comment by afebbraro** [comment]
waiting on PR with new CSS. Putting in "needs changes" till then.
